src/ApparentHorizons/ChangeCenterOfStrahlkorper.hpp

```diff
@@ -0,0 +1,24 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <array>
+
+#include "ApparentHorizons/Strahlkorper.hpp"
+#include "Utilities/Gsl.hpp"
+
+/// Changes the expansion center of a Strahlkorper.
+template <typename Frame>
+void change_expansion_center_of_strahlkorper(
+    gsl::not_null<Strahlkorper<Frame>*> strahlkorper,
+    const std::array<double, 3>& new_center) noexcept;
+
+/// Changes the expansion center of a Strahlkorper to the physical
+/// center.  This is an iterative algorithm that ends when the
+/// expansion center equals the physical center to the given
+/// tolerance.
+template <typename Frame>
```

Comment:
In practice, how much does the tolerance change under different runs we might do? Maybe recommend or set a good default if this doesn't need to change a lot depending on the run? Does spec do something meaningful?

I ask because I'm worried about proliferation of thresholds in general

Reply:
I got rid of the tolerance, made it roundoff.  It's not very expensive to do so.